expirationd.lua

```diff
@@ -580,6 +541,8 @@ end
 -- params:
 --   name -- task's name
 local function expirationd_task_stats(name)
+    checks('string')
```

Comment:
name is an optional argument. See code below. If name is nil then stats for all tasks is returned.

Reply:
make it optional